models/virtual_service.go

```diff
@@ -56,3 +56,18 @@ func (vService *VirtualService) Parse(virtualService kubernetes.IstioObject) {
 	vService.Tcp = virtualService.GetSpec()["tcp"]
 	vService.Tls = virtualService.GetSpec()["tls"]
 }
+
+// IsVirtualService returns true if VirtualService hosts applies to the service
+func (vService *VirtualService) IsVirtualService(namespace string, serviceName string) bool {
```

Comment:
This name is misleading.
A VirtualService model is self invoking if IsVirtualService() ? It doesn't sound good to me; instead this method is validating if a service name is part of the hosts.
Suggestion IsValidHost() sounds better.

Reply:
Also, this name can be propagated into the Test naming strategy as well.

Reply:
Funny, I wasn't really happy with this name either, I will change it.